assets/pages/contributions-landing/contributionsLanding.scss

```diff
@@ -356,8 +357,9 @@ strong {
 	}
 
 	.component-bundle__content-intro {
-		line-height:20px;
-		margin-bottom:24px;
+		line-height: 20px;
+		margin-bottom: 24px;
```

Comment:
Probably using the variables would be better:`$gu-v-spacing` * 2

Reply:
Haha, yeah ok.